skimage/feature/brief.py

```diff
@@ -130,10 +134,10 @@ def __init__(self, descriptor_size=256, patch_size=49,
         self.mode = mode
         self.sigma = sigma
 
-        if seed is None:
-            self.seed = np.random.SeedSequence().entropy
+        if rng is None:
+            self.seed = np.random.SeedSequence()
         else:
-            self.seed = seed
+            self.seed = rng
```

Comment:
Is this needed? Shouldn't it just be
```
self.seed = rng
```
Also, should it be `self.rng`?

Reply:
No, because the rng changes as it gets used. The idea here is to re-seed every time `.extract` is called.

Reply:
I think this is correct. You want to re-use the same seed every time you run `.extract`. You can only do that if the seed is set to something other than None. So, if the RNG is None, then you grab a random seed once, and use it on every subsequent call of `.extract`.